lib/pkcs7.js

```diff
@@ -150,6 +150,13 @@ p7.createSignedData = function() {
       }
 
       // TODO: parse crls
+
+      msg.contentInfo = msg.rawCapture.contentInfo;
+      msg.signerInfos = msg.rawCapture.signerInfos;
+
+      if (msg.signerInfos) {
```

Comment:
Please remove space after `if`.  I apologize for the sorry state of our linting in this module.  We have eslint rules in place, but no one has fixed all the existing linting errors that existed prior to adding the rules.